README.md

```diff
@@ -113,8 +113,7 @@ Instantiate an **AlchemyLanguage** object and set its api key via a **TokenAuthe
 
 ```swift
 
-let token = TokenAuthenticationStrategy(token: <API_KEY>)
-let alchemyLanguageInstance = AlchemyLanguage(tokenAuthenticationStrategy: token)
+let alchemyLanguage = AlchemyLanguage(apiKey: String)
```

Comment:
Please update (apiKey: String) to (apiKey: "your-apikey-here") then I will merge in PR.

Thanks,
-veh